### PR TITLE
Feature/add connect supplier widget

### PR DESCRIPTION
**Pull request by lasong**

### CODE REVIEW

File: src/server/api/inChannelContract.js [hunk 1]
```diff
@@ -36,6 +36,15 @@ module.exports.get = function(customerId, supplierId)
     });
 }
 
+module.exports.allForCustomer = function(customerId)
+{
+    return this.db.models.InChannelContract.findAll(
+        {where: {customerId: customerId}, order: [['changedOn', 'ASC']]})
+    .then((data) => {
+        return data;
```
Comment (kwierchris) [inline src/server/api/inChannelContract.js]: Does not make sense. Useless ^^

File: src/server/routes/index.js [hunk 2]
```diff
@@ -93,6 +93,7 @@ module.exports.init = function(app, db, config)
         // InChannelContract
         // TODO: Create own express Router
         //
+        app.get('/api/config/inchannelcontracts/:customerId', (req, res) => this.sendInChannelContractsForCustomer(req, res));
```
Comment (kwierchris) [inline src/server/routes/index.js]: Change :customerId to :tenantId.
:tenantId will be s_${supplierId} or c_$(customerId}. Depending on which one is passed, the data of the other entity is displayed. E.g. s_${supplierId} leads to a query "where supplierId = ${supplierId}" to display all data for associated customers.

File: src/server/routes/index.js [hunk 3]
```diff
@@ -592,6 +593,19 @@ console.log(">> sendInChannelContract - businesspartner: ", bp.supplierId, bp.cu
     }
 }
 
+module.exports.sendInChannelContractsForCustomer = function(req, res)
```
Comment (kwierchris) [inline src/server/routes/index.js]: Change it according to the comment on the endpoint.

File: src/server/routes/index.js [hunk 4]
```diff
@@ -592,6 +593,19 @@ console.log(">> sendInChannelContract - businesspartner: ", bp.supplierId, bp.cu
     }
 }
 
+module.exports.sendInChannelContractsForCustomer = function(req, res)
+{
+    const customerId = req.params.customerId;
+    if (process.env.NODE_ENV === 'development' || req.opuscapita.userData('customerId') === customerId) {
```
Comment (kwierchris) [inline src/server/routes/index.js]: Remove custom security check. ACL will cover that.

Comment (lasong) [inline src/server/routes/index.js]: 👍

**Pull request merged**